src/get_spixel.F90

```diff
@@ -342,6 +354,16 @@ subroutine Get_SPixel(Ctrl, SAD_Chan, SAD_LUT, MSI_Data, RTM, SPixel, status)
    call Get_X(Ctrl, SPixel, MSI_Data, status)
 !  if (status /= 0) go to 99 ! Skip further data reading
 
+   ! It seems that it is possible that an invalid pixel, with no
+   ! defined measurement, can reach this point without causing a
+   ! SPixel status. Check for this condition here
+   if (SPixel%Ind%Ny == 0) then
```

Comment:
While I'm not surprised this is possible, someone should track down what the failure mode is and fix that too.